src/main/java/frc/robot/commands/RatchetingMechanism.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ServoSubsystem;
+
+public class RatchetingMechanism extends Command {
+  private final ServoSubsystem servo;
+  public RatchetingMechanism(ServoSubsystem servo) {
+    this.servo = servo;
+    addRequirements(servo);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    if (servo.get_Servo() == 0.0) {
+      servo.set_Servo(1.0);
+    } else if (servo.get_Servo() == 1.0) {
+      servo.set_Servo(0.0);
```

Comment:
These servo positions should be written in the `Constants` class.

As this logic is written, this means this command will do nothing when the servo is not at positions 0.0 or 1.0. It may be useful to also handle the conditions where the servo is between 0 and 1.